testsuite/MDAnalysisTests/transformations/test_rotate.py

```diff
@@ -28,29 +28,29 @@
 from numpy.testing import assert_array_almost_equal
 
 import MDAnalysis as mda
-from MDAnalysis.transformations import rotate
+from MDAnalysis.transformations import rotateby
 from MDAnalysis.lib.transformations import rotation_matrix
 from MDAnalysisTests.datafiles import GRO
 
-def test_translate_custom_position():
+def test_rotate_custom_position():
     u = mda.Universe(GRO)
     vector = np.float32([1,0,0])
     pos = np.float32([0,0,0])
     matrix = rotation_matrix(math.pi/2, vector, pos)[:3, :3]
     ref = u.trajectory.ts
     ref.positions = np.dot(ref.positions, matrix)
-    transformed = rotate(math.pi/2, vector, position = pos)(ref) 
+    transformed = rotateby(math.pi/2, vector, position = pos)(ref) 
```

Comment:
The conventions in PEP8 say that there should not be spaces around the `=` here, but there should be spaces around the `/`.